server/parsec/components/account.py

```diff
@@ -27,3 +32,20 @@ async def test_new_account(self) -> str:
         assert await self.create(email) is None
 
         return email
+
+    @api
+    async def api_account_send_email_validation_token(
+        self,
+        client_ctx: AnonymousAccountClientContext,
+        req: anonymous_account_cmds.latest.account_send_email_validation_token.Req,
+    ) -> anonymous_account_cmds.latest.account_send_email_validation_token.Rep:
+        outcome = await self.create(req.email)
+        match outcome:
+            case CreateAccountBadOutcome.INVALID_EMAIL:
+                return anonymous_account_cmds.latest.account_send_email_validation_token.RepInvalidEmail()
+            case CreateAccountBadOutcome.ACCOUNT_ALREADY_EXISTS:
+                # Do nothing
```

Comment:
```suggestion
                # Respond OK without sending token to prevent creating 
                # an oracle of email addresses registered in the service.
```